osprey-mock-service.js

```diff
@@ -82,16 +96,18 @@ function handler (method) {
   var statusCode = getStatusCode(method)
   var response = (method.responses || {})[statusCode] || {}
   var bodies = response.body || {}
-  var headers = response.headers || method.headers || {}
+  var headers = {}
   var types = Object.keys(bodies)
 
   // Set up the default response headers.
-  Object.keys(headers).forEach(function (key) {
-    var value = headers[key]
-    if (value && value.default) {
-      headers[key] = value.default
-    }
-  })
+  if (Array.isArray(response.headers)) {
+    response.headers.forEach(function (header) {
+      if (header && (header.default || header.example || header.examples)) {
+        var example = getExampleValueFromHeader(header)
+        headers[header.name] = header.default || example
```

Comment:
No, this should be reusable since it will be used to also get a random example from bodies. It's the same logic. We discussed that before.